### ISSUE TITLE
bot marks issues as stale without notifying the author beforehand

### ISSUE THREAD

**Issue by DetachHead** [eslint-community/eslint-plugin-eslint-comments#164]
i don't like stale bots as they often close issues that are still valid just because they haven't been commented on in a few weeks. but if you feel it's necessary, i think it should first comment informing the author that the issue will be closed when it's first marked as stale. otherwise, there's no way for the author to know that the bot is waiting for a reply:

![image](https://github.com/eslint-community/eslint-plugin-eslint-comments/assets/57028336/176878c9-d892-4694-be95-b31c69fb6d16)

(from https://github.com/eslint-community/eslint-plugin-eslint-comments/issues/130#issuecomment-1865419778)

this would make it consistent with how stale bots work in other repos

**Comment by GrantGryczan** [comment]
Agreed, and I think #136 should be reopened as it's still an issue and I still care about it (despite that I'm not the one who reported it).

**Comment by FloEdelmann** [comment]
For quick reference: Here are all issues that were closed as stale: https://github.com/eslint-community/eslint-plugin-eslint-comments/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed+label%3AStale

**Comment by Samuel-Therrien-Beslogic** [comment]
Please re-open https://github.com/eslint-community/eslint-plugin-eslint-comments/issues/162#issuecomment-1963096815 and https://github.com/eslint-community/eslint-plugin-eslint-comments/issues/161#issuecomment-1963020327
I was never even notified of the stale tag and 1 week is way too short.

**Comment by GrantGryczan** [comment]
What about the issues that were closed as stale? May they be reopened please?
